spec/presenters/summary/sections/income_details_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe Summary::Sections::IncomeDetails do
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) do
+    instance_double(
+      CrimeApplication,
+      to_param: '12345',
+      income: income,
+    )
+  end
+
+  let(:income) do
+    instance_double(
+      Income,
+      income_above_threshold: 'yes'
+    )
+  end
+
+  describe '#name' do
+    it { expect(subject.name).to eq(:income_details) }
+  end
+
+  describe '#show?' do
+    context 'when there is no income_details' do
+      let(:income) { nil }
+
+      it 'shows this section' do
+        expect(subject.show?).to be(false)
+      end
```

Comment:
Same here - should the test be `it 'does not show this section'` ?

Reply:
I renamed AND reordered (just FYI in case it causes confusion)